examples/ui-demo/src/app/providers.tsx

```diff
@@ -11,6 +12,13 @@ const alchemyConfig = createConfig({
   rpcUrl: "/api/rpc",
   chain: sepolia,
   ssr: true,
+  connectors: [
```

Comment:
we should probs also add `injected` here

Reply:
so I recommend not adding 'injected' manually here since it's outdated (see the notion doc) - wagmi will automatically detect available injected providers through eip6963 and insert it into the list of connectors

Reply:
Oh yea that makes sense, I was testing on a browser that I thought had injected wallets and it didn't lol

How do we control the order here? I would expect the injected ones to show up first
![image](https://github.com/alchemyplatform/aa-sdk/assets/4642570/d485eca1-ba81-45bf-95ce-3c546baf31e6)